authentication/securestore/main.go

```diff
@@ -207,6 +210,55 @@ func ValidateBackend(backend string) error {
 	return nil
 }
 
+func keychainGet() ([]byte, error) {
+	credentials, err := gokeyring.Get(ItemKey, ItemKey)
+	if err != nil && err != gokeyring.ErrNotFound {
+		return []byte(credentials), err
+	} else if err == gokeyring.ErrNotFound {
+		return []byte(""), nil
+	}
+
+	if len(credentials) == 0 {
+		// No creds to decompress, return early
+		return []byte(""), nil
+	}
+
+	creds, err := decompressConfig([]byte(credentials))
+	if err != nil {
+		return nil, err
+	}
+	return creds, nil
+}
+
+func keychainUpsert(creds []byte) error {
+	compressed, err := compressConfig(creds)
+	if err != nil {
+		return err
+	}
+
+	err = gokeyring.Set(ItemKey, ItemKey, string(compressed))
```

Comment:
According to the docs, the second argument here is the user:

```
// Set password in keyring for user.
func Set(service, user, password string) error
```

I wonder if this will cause issues in case there's more than one user

Reply:
We are setting user as the same Item Key to maintain parity with the way configuration is handled today. If a user logs in with a different account, it will just overwrite what is in the keyring with the new info.

Reply:
![image](https://github.com/openshift-online/ocm-sdk-go/assets/5939777/f08cf098-7f2a-4194-93ab-9bedba0a54b8)